template/devshell.toml

```diff
@@ -0,0 +1,2 @@
+[[commands]]
```

Comment:
Can you also add a comment at the top that links to the project?

The idea is for users that don't know about the project to quickly be able to figure out what it is about.